src/libraries/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -126,6 +126,8 @@ public abstract partial class DSA : System.Security.Cryptography.AsymmetricAlgor
         public static System.Security.Cryptography.DSA Create(System.Security.Cryptography.DSAParameters parameters) { throw null; }
         public static new System.Security.Cryptography.DSA Create(string algName) { throw null; }
         public abstract byte[] CreateSignature(byte[] rgbHash);
+        public byte[] CreateSignature(byte[] rgbHash, System.Security.Cryptography.DSASignatureFormat signatureFormat) { throw null; }
```

Comment:
@krwq this PR is adding new public APIs. Please make sure to document them with triple slash in their implementation so we can port them in the future to dotnet-api-docs.

cc @bartonjs in case you'll follow up with this PR.